script/deploy/facets/DeployStandardizedCallFacet.s.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+import { DeployScriptBase } from "./utils/DeployScriptBase.sol";
+import { stdJson } from "forge-std/Script.sol";
+import { StandardizedCallFacet } from "lifi/Facets/StandardizedCallFacet.sol";
+
+contract DeployScript is DeployScriptBase {
+    using stdJson for string;
+
+    constructor() DeployScriptBase("StandardizedCallFacetFacet") {}
+
+    function run()
+        public
+        returns (StandardizedCallFacet deployed, bytes memory constructorArgs)
+    {
+        string memory path = string.concat(
+            vm.projectRoot(),
+            "/config/standardizedCallFacet.json"
+        );
+        string memory json = vm.readFile(path);
+        address example = json.readAddress(
```

Comment:
This seems to be erroneously used from the template to me. 
The facet does not have a constructor and the whole example variable should be removed imho (same for line 26)